topdown/trace.go

```diff
@@ -196,6 +206,24 @@ func formatEventSpaces(event *Event, depth int) int {
 	return depth + 1
 }
 
+func formatLocation(event *Event) string {
+	if event.Op == NoteOp {
+		return fmt.Sprintf("%-19v", "note")
+	}
+
+	location := event.Location
+	if location == nil {
+		// Most likely a trace event
+		return fmt.Sprintf("%-19v", "trace")
+	}
+
+	if location.File == "" {
+		return fmt.Sprintf("%-19v", fmt.Sprintf("%.15v:%v", "buffer", location.Row))
```

Comment:
nit: Instead of `buffer` could we use `query`?

Reply:
Yep, query does indeed make more sense then buffer. I updated the PR.